go-controller/pkg/node/node.go

```diff
@@ -99,6 +99,67 @@ func setupOVNNode(node *kapi.Node) error {
 			return fmt.Errorf("error setting OVS encap-port: %v\n  %q", errSet, stderr)
 		}
 	}
+	if config.Monitoring.NetFlowTargets != "" {
+		collectors := ""
+		for _, v := range strings.Split(config.Monitoring.NetFlowTargets, ",") {
+			collectors += "\"" + v + "\"" + ","
+		}
+		collectors = strings.TrimSuffix(collectors, ",")
+
+		_, stderr, err := util.RunOVSVsctl(
+			"--",
+			"--id=@netflow",
+			"create",
+			"netflow",
+			fmt.Sprintf("targets=[%s]", collectors),
+			"--",
+			"set", "bridge", "br-int", "netflow=@netflow",
+		)
+		if err != nil {
+			return fmt.Errorf("error setting NetFlow: %v\n  %q", err, stderr)
+		}
+	}
+	if config.Monitoring.SFlowTargets != "" {
+		collectors := ""
+		for _, v := range strings.Split(config.Monitoring.SFlowTargets, ",") {
+			collectors += "\"" + v + "\"" + ","
+		}
+		collectors = strings.TrimSuffix(collectors, ",")
+
+		_, stderr, err := util.RunOVSVsctl(
+			"--",
+			"--id=@sflow",
+			"create",
+			"sflow",
+			"agent=ovn-k8s-mp0",
+			fmt.Sprintf("targets=[%s]", collectors),
+			"--",
+			"set", "bridge", "br-int", "sflow=@sflow",
+		)
+		if err != nil {
+			return fmt.Errorf("error setting SFlow: %v\n  %q", err, stderr)
+		}
+	}
+	if config.Monitoring.IPFIXTargets != "" {
+		collectors := ""
+		for _, v := range strings.Split(config.Monitoring.IPFIXTargets, ",") {
+			collectors += "\"" + v + "\"" + ","
+		}
+		collectors = strings.TrimSuffix(collectors, ",")
+
+		_, stderr, err := util.RunOVSVsctl(
+			"--",
+			"--id=@ipfix",
+			"create",
+			"ipfix",
+			fmt.Sprintf("targets=[%s]", collectors),
+			"--",
+			"set", "bridge", "br-int", "ipfix=@ipfix",
+		)
```

Comment:
I don't see any timeout or aging setting for the for the above ovs config , do you know if ovs support aging to achieve LRU alg. who manage old flows and remove them ?

Reply:
I don't know of any timeout setting for netflow, you can check the ovs source for that, maybe there is, not in docs at least.

Reply:
```
ovs−vsctl set netflow br-int active_timeout=<value in seconds>
```

Reply:
ah, the active timeout, I didn't know what you meant by aging.
We decided to go light on over-configuring this feature, leave it opened for the future. You can see more context in the openshift enhancement where this was discussed, openshift/enhancements repo /network/netflow.

Reply:
why is there a timeout specified for netflow, but not for slfow or ipfix?

Reply:
Yeah, I missed adding the corresponding flag for ipfix.
There is no such timeout flag on sFlow.